.github/workflows/build.yml

```diff
@@ -0,0 +1,71 @@
+---
+name: Builder
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  build:
+    name: Build
+    strategy:
+      matrix:
+        distro:
+          - 'alpine:3.11'
+          - 'alpine:3.10'
+          - 'alpine:3.9'
+          - 'archlinux:latest'
+            #- 'centos:8'
+          - 'centos:7'
+          - 'centos:6'
+          - 'debian:bullseye'
+          - 'debian:buster'
+          - 'debian:stretch'
+          - 'fedora:31'
+          - 'fedora:30'
+          - 'fedora:29'
+            #- 'opensuse/leap:15.2'
+            #- 'opensuse/leap:15.1'
+            #- 'opensuse/leap:15.0'
+            #- 'opensuse/tumbleweed:latest'
```

Comment:
Is there some specific reason we're not handling OpenSUSE here?

Reply:
It is broken for some reason. I will re-test later once merged to master and uncomment.